handlers/router.go

```diff
@@ -20,8 +26,15 @@ func NewRouter() Router {
 
 // Route call the appropriate handler for a request based on its path
 func (r *router) Route(request events.APIGatewayProxyRequest) (interface{}, int) {
-	if request.Path == "/hello" {
-		return helloWorld(request)
+	routes := []RouteHandler{
+		newHelloWorld(),
```

Comment:
Additional handlers would be added to this array

Reply:
Would this be better put in an `AddRoute` function that could be called from the main
Routes could then be a map in `router`, and `Route` would then loop through the routes on router

This would make it easier to:
- test
- set it up with different routes from the main (if we ever made config files or anything to determine which routes to use, they could all be read from the main
- We would only initialise `routes` once instead of on every call to `Route`